packages/react/src/lib/accessibility/Behaviors/Tree/treeBehavior.ts

```diff
@@ -0,0 +1,39 @@
+import * as keyboardKey from 'keyboard-key'
+import { Accessibility, AccessibilityAttributes, FocusZoneMode } from '../../types'
+import { FocusZoneDirection } from '../../FocusZone'
+
+/**
+ * @specification
+ * Adds role 'tree' to 'root' slot.
+ * Adds attribute 'aria-labelledby' based on the property 'aria-labelledby' to 'root' slot.
+ * Embeds component into FocusZone.
+ * Provides arrow key navigation in vertical direction.
+ * Triggers 'expandSiblings' action with '*' on 'root'.
+ */
+const treeBehavior: Accessibility<TreeBehaviorProps> = props => {
+  return {
+    attributes: {
+      root: {
+        role: 'tree',
+        'aria-labelledby': props['aria-labelledby'],
+      },
+    },
+    keyActions: {
+      root: {
+        expandSiblings: {
+          keyCombinations: [{ keyCode: keyboardKey['*'] }],
```

Comment:
Does this mean that the siblings are expanded no matter which key is pressed?

Reply:
That means that all siblings are expanded when `*` is pressed. But ARIA design pattern